modules/rosa-edit-objects.adoc

```diff
@@ -132,6 +154,13 @@ Update the default ingress using the sub-domain identifier `apps` on a cluster n
 $ rosa edit ingress --private=false --cluster=mycluster apps
 ----
 
+Update the load balancer type of the apps2 ingress.
```

Comment:
I am not familiar with this content area but should `apps2` be in backticks?? 

Also, I am curious about ingress. I haven't really seen it lowercase anywhere.